autoload/neomake/makers/ft/c.vim

```diff
@@ -1,11 +1,17 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#c#EnabledMakers()
+    let makers = []
```

Comment:
The special requirements are that `clang-tidy` should only be added if `clang` and `clang-tidy` are available on the system. Since both `clang` and `clang-tidy` makers can coexist (they achieve different objectives), I do the relevant tests with some `neomake#utils#Exists` calls right after this.